src/wallet/wallet2.cpp

```diff
@@ -3085,6 +3115,29 @@ crypto::hash8 wallet2::get_short_payment_id(const pending_tx &ptx) const
 void wallet2::commit_tx(pending_tx& ptx)
 {
   using namespace cryptonote;
+  
+  if(m_light_wallet) {
+    
+    cryptonote::COMMAND_RPC_SUBMIT_RAW_TX::request oreq;
+    cryptonote::COMMAND_RPC_SUBMIT_RAW_TX::response ores;
+    
+    oreq.address = get_account().get_public_address_str(m_testnet);
+    oreq.view_key = string_tools::pod_to_hex(get_account().get_keys().m_view_secret_key);
+    oreq.tx = epee::string_tools::buff_to_hex_nodelimer(tx_to_blob(ptx.tx));
+    m_daemon_rpc_mutex.lock();
+    bool r = epee::net_utils::invoke_http_json("/submit_raw_tx", oreq, ores, m_http_client, rpc_timeout, "POST");
+    m_daemon_rpc_mutex.unlock();
+    THROW_WALLET_EXCEPTION_IF(!r, error::no_connection_to_daemon, "submit_raw_tx");
+    THROW_WALLET_EXCEPTION_IF(ores.status != "OK", error::tx_rejected, ptx.tx, ores.status, ores.error);
```

Comment:
Getting really confused here. I didn't push anything after you commented on this, but it seems like you commented on an old version? Anyway, throw_on_rpc_response_error doesn't work because openmonero and mymonero use different status messages. Link to current version:
https://github.com/monero-project/monero/pull/2109/files#diff-9d580668dab930045943902048e2ac22R3163